layers/+tools/shell/funcs.el

```diff
@@ -27,6 +27,14 @@
   (let ((default-directory (projectile-project-root)))
     (call-interactively 'spacemacs/default-pop-shell)))
 
+(defun spacemacs/projectile-run-shell ()
+  "Run the default shell in the project root directory."
+  (interactive)
+  (let ((shell (cl-case shell-default-shell
+                 ('multi-term #'projectile-multi-term-in-root)
+                 (t (intern (format "projectile-run-%S" shell-default-shell))))))
```

Comment:
When `shell-default-shell` is `ansi-term`, it would fail.
Just add `('ansi-term nil)`
And change `let` to `when-let`